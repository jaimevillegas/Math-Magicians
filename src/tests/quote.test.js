import { render, screen } from '@testing-library/react';
import Quote from '../components/quote';

describe('Quote test', () => {
  test('Quote component display loading status', () => {
    render(<Quote />);
    const quote = screen.getByTitle('quote-container');
    expect(quote).toMatchSnapshot();
  });
});