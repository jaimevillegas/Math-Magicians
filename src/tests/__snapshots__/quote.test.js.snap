// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote test Quote component display loading status 1`] = `
<div
  class="quote-container"
  title="quote-container"
>
  Loading Quote
</div>
`;
